Source/DafnyLanguageServer.Test/Various/PluginsTest.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
```

Comment:
This test-class has lots of similarities to `PluginsAdvancedTest.cs`. It might be handy to share the common parts between these files.